ddtrace/mocktracer/mocktracer.go

```diff
@@ -57,13 +58,13 @@ type Tracer interface {
 // to activate the mock tracer. When your test runs, use the returned
 // interface to query the tracer's state.
 func Start() Tracer {
-	t := newMockTracer()
-	old := tracer.GetGlobalTracer()
-	if _, ok := old.(*mocktracer); ok {
-		tracer.StopTestTracer()
-	}
-	tracer.SetGlobalTracer(t)
-	return t
+	var t tracer.Tracer = newMockTracer()
+	internal.SetGlobalTracer(t)
```

Comment:
`internal.SetGlobalTracer` swaps and stops the previously set tracer, so it is functionally equal.